python/pyarrow/types.pxi

```diff
@@ -926,13 +926,17 @@ cdef class Schema:
         """
         Append a field at the end of the schema.
 
+        In contract to Python's ``list.append()`` it does return a new
```

Comment:
`contrast`? But that sounds odd to my ear.

Reply:
Ah, yes, that's a typo, I meant 'contrast'. But do you still find that sound odd to your ear? (non-native english speaker here, it's what translate gives and sounds good, but happy to use something else)

Reply:
It also sounds good in French, @bkietz ^